readyapi4j-core/src/main/java/com/smartbear/readyapi4j/support/ContentUtils.java

```diff
@@ -0,0 +1,46 @@
+package com.smartbear.readyapi4j.support;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.dataformat.xml.XmlMapper;
+import io.swagger.util.Json;
+import io.swagger.util.Yaml;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ContentUtils {
+
+    private final static Logger LOG = LoggerFactory.getLogger(ContentUtils.class);
+
+    public static String serializeContent(Object requestBody, String mediaType) {
```

Comment:
We should add javadoc and probably mention what media types are supported. OR we can have an enum for media types, so user knows what exactly is supported.